man/sysusers.d.xml

```diff
@@ -209,37 +210,45 @@ u      root   0               "Superuser"           /root</programlisting>
     <refsect2>
       <title>GECOS</title>
 
-      <para>A short, descriptive string for users to be created,
-      enclosed in quotation marks. Note that this field may not
-      contain colons.</para>
+      <para>A short, descriptive string for users to be created, enclosed in
+      quotation marks. Note that this field may not contain colons.</para>
 
-      <para>Only applies to lines of type <varname>u</varname> and
-      should otherwise be left unset, or be set to
-      <literal>-</literal>.</para>
+      <para>Only applies to lines of type <varname>u</varname> and should otherwise
+      be left unset (or <literal>-</literal>).</para>
     </refsect2>
 
     <refsect2>
       <title>Home Directory</title>
 
-      <para>The home directory for a new system user. If omitted,
-      defaults to the root directory. It is recommended to not
-      unnecessarily specify home directories for system users, unless
-      software strictly requires one to be set.</para>
+      <para>The home directory for a new system user. If omitted, defaults to the
+      root directory.</para>
 
-      <para>Only applies to lines of type <varname>u</varname> and
-      should otherwise be left unset, or be set to
-      <literal>-</literal>.</para>
+      <para>Only applies to lines of type <varname>u</varname> and should otherwise
+      be left unset (or <literal>-</literal>). It is recommended to omit this, unless
+      software strictly requires the home directory to be set.</para>
+    </refsect2>
+
+    <refsect2>
+      <title>Shell</title>
+
+      <para>The login shell of the user. If not specified, this will be set to
+      <filename>/sbin/nologin</filename>, except for the UID of the user is 0, in
+      which case <filename>/bin/sh</filename> will be used.</para>
+
+      <para>Only applies to lines of type <varname>u</varname> and should otherwise
+      be left unset (or <literal>-</literal>). It is recommended to omit this, unless
+      a shell different <filename>/sbin/nologin</filename> must be used.</para>
```

Comment:
hmm, so, if /bin/false and /sbin/nologin are indeed two different things (as @sgallagher suggested), and the difference is relevant, then maybe we should add a quick mention of this here, and explain what's going on? already, so that I myself can remember this? ;-) But mostly because this appears to be a relevant distinction for system users, and we are about system user management here...

Reply:
I read a bunch of the stackoverflow threads, and the only distinction I find is that nologin prints a message. Some ftp servers might disallow or might have disallowed access if the shell is not in /etc/shells, and /sbin/nologin usually is and /bin/false usually not, but well, when somebody testing this, it turned to be untrue, and anyway, I don't think we care about ftp. So in the end it seems `/sbin/nologin` should be used because it prints a message and logs, I don't see any particular reason for `/bin/false`.

Reply:
ok, very good